sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CastSuite.scala

```diff
@@ -727,6 +727,19 @@ class CastSuite extends SparkFunSuite with ExpressionEvalHelper {
     }
   }
 
+  test("cast struct with a timestamp field") {
+    val origSchema = StructType(
+      Seq(StructField( "tsField", TimestampType, false ))
+    )
+    val trgSchema = StructType(
+      // nine out of ten times I'm casting a struct, it's to normalize its fields nullability
+      Seq(StructField( "tsField", TimestampType, true ))
```

Comment:
nit: remove extra whitespaces and use `nullable = true` for 3rd argument of `StructField`.